src/main/java/graphql/annotations/GraphQLAnnotations.java

```diff
@@ -571,7 +574,7 @@ protected GraphQLFieldDefinition getField(Method method) throws GraphQLAnnotatio
             List<GraphQLFieldDefinition> fieldDefinitions = outputType instanceof GraphQLObjectType ?
                     ((GraphQLObjectType) outputType).getFieldDefinitions() :
                     ((GraphQLInterfaceType) outputType).getFieldDefinitions();
-            relay = RELAY_TYPES.mutationWithClientMutationId(title, method.getName(),
+            relay = GraphQLAnnotations.this.relay.mutationWithClientMutationId(title, method.getName(),
```

Comment:
Please remove the ``GraphQLAnnotations.this``

Reply:
done